code-challenges.test.js

```diff
@@ -15,48 +15,135 @@
 // Pseudo code:
 
 // a) Create a test with expect statements for each of the variables provided.
-
+// Answered by Chris in class
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
+// Answered by Chris in class. 
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // Pseudo code:
+// Input: An object as an argument
+// Output: If the number inside it is evenly divisible by three or not
+// Create a function called divByThree that passes the object as an argument
+// Use a ternary to decide if a number modulus 3 is === 0, if yes then the number is evenly divisible, if not then the number is not
+// Retrun the expected statements
+
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("divByThree", (object) => {
+    it("decides if the number inside it is evenly divisible by three or not.", () => {
+        expect(divByThree(object1)).toEqual("15 is divisible by three")
+        expect(divByThree(object2)).toEqual("0 is divisible by three")
+        expect(divByThree(object3)).toEqual("-7 is not divisible by three")
+    })
+})
+
+// FAIL  ./code-challenges.test.js
+// divByThree
+//   ✕ decides if the number inside it is evenly divisible by three or not.
+
+// ● divByThree › decides if the number inside it is evenly divisible by three or not.
+
+//   ReferenceError: divByThree is not defined
+
+
+// b) Create the function that makes the test pass.
 const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
 const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+const divByThree = (obj) => {
+    return obj.number % 3 === 0 ? `${obj.number} is divisible by three` : `${obj.number} is not divisible by three`
+}
+
+// PASS  ./code-challenges.test.js
+// divByThree
+//   ✓ decides if the number inside it is evenly divisible by three or not. (1 ms)
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // Pseudo code:
+// Input: An array of words
+// Output: An array with all the words capitalized
+// Create a function name capArr
+// Iterate through the array with a .map , use str.charAt(0) to access the first letter of each string, then use .toUpperCase() to capitalize that character while + str.slice() gives us a stopping point so not all letters get capitalized. 
```

Comment:
Very close. The `+ str.slice()` is not giving a stopping point for the uppercasing, but it is concatenating (the `+` operator) the newly capitalized first letter of the string with the rest of the string. `.slice` returns a portion of the string, determined by the arguments you give it. Since you gave it an argument of 1, it is returning a substring of the original string starting at index 1 to the end of the string, which is what is getting added to the newly capitalized letter.